docs/GAME.md

```diff
@@ -1,116 +1,136 @@
 
 ##Agent Snooping
+
 The goal is to simulate the way the real world works.  There are a few places where this is difficult because there is a disparity between what is technically possible and what is realistically possible in the real world.  It boils down to this: The game involves 8 players, the world has 8 billion.
 
 For example, in a world of 8 players it is technically possible to tell not only who is using Tor, but when they send out messages.  In a land of 8 billion you wouldn't be tracked so closely unless there was reason to.  In a real world land of 8 you could wiretap everyone with minimal cost.
 
 It is possible that the agent should only be allowed to wiretap one person at a time to get the explicit details on them?
 
-This section describes what COULD be tracked by an agent.  It will be pruned down to what we are actually going to track as we think it through.
+This section describes how the various forms of communication can be tracked by an agent.
 
 KEY: 
- {+} means the agent gets this information regardless of wiretapping
+ {+} means something good for the agent, without agent action.
  
- {*} means the agent gets this information only when wiretapped
+ {*} means the agent gets this information only when the agent has gained an explicit advantage.
  
- {-} means it is a way for the user to fight against the agent
+ {-} means something the activists can use to fight against the agent
 
 ###Tor
+
  {+} The agent knows who has enabled Tor.
- 
- {-} ... unless a player is using a private Tor bridge, in which case the agent know nothing.
 
- {*} ... unless the player is wiretapped, in which case the agent still knows the player is using Tor.
+ {+} Citizens can't use Tor unless they have been trained.
+
+ {-} The journalist starts the game trained to use Tor.
 
+ {-} Anybody who has been trained to use Tor can train another person to use Tor by sending instructions.
+ 
 ###Email
+
  {+}  The agent can read the content of all emails.
 
  {-} ... unless the sender uses PGP encryption (the agent will still see the content, but it will be encrypted.)
 
- {+}  The agent can therefore tell when a specific account has sent an email.
+ {+}  The agent can tell when an email account has sent an email.
 
- {+}  The agent can also therefore view the destination (to, from, cc, bcc) of all emails.
+ {+}  The agent can view the destination (to, from, cc, bcc) of all emails.
 
  {+}  The agent knows which email accounts a player is accessing.
 
- {-} ... unless the player is using SSL or Tor.
+ {+}  The agent knows which emails have been read, and when.
+
+ {-} ... unless the player is using SSL.
+
+ {+} ... until the agent supoenas the Email server, at which point SSL no longer has any effect.
```

Comment:
nitpicky: `subpoena` (this typo is elsewhere in the document as well.)

less nitpicky: How does an agent subpoena these servers? I think we talked about making it scheduled, in which case it may help if we included a timeline in the rules.

```
Turn 1: Game begins
Turn 3: Email server is subpoena'd
...
```